app/models/concerns/last_menstrual_period_measureable.rb

```diff
@@ -30,7 +30,11 @@ def _last_menstrual_period_on_date(date)
     return nil unless initial_call_date && last_menstrual_period_weeks
     weeks = 7 * (last_menstrual_period_weeks || 0)
     days = (last_menstrual_period_days || 0)
-    (date - initial_call_date) + weeks + days
+    if (date - patient.initial_call_date) + weeks + days > 280
```

Comment:
Hey, looks like the test failures are legit, and I think the cause of them is the use of `patient` here. Since these methods in this concern are essentially methods on `patient`, it's trying to hit `patient.patient.initial_call_date` instead of `patient.initial_call_date` and choking.

tldr: I think you can remove the word `patient` from `patient.initial_call_date` here and below and it'll fly.